prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,14 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log('languages array', languages);
+
+const firstElement = languages[0];
+console.log('the first element of the array is', firstElement);
+
+const thirdElement = languages[2];
+console.log('the 3rd element of the array is', thirdElement);
+
+const length = languages.length;
+console.log('The length of the array is', length);
+
+const lastElement = languages[5];
```

Comment:
you want to calculate this using the length variable above. you want to avoid using hard coded values like `5`